Packs/XDRRetroHunt/Playbooks/playbook-XDR_Retro_Hunt_-_Query_README.md

```diff
@@ -0,0 +1,34 @@
+
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+This playbook does not use any integrations.
+
+### Scripts
+This playbook does not use any scripts.
+
+### Commands
+* xdr-xql-file-event-query
+* xdr-xql-generic-query
+* xdr-xql-dns-query
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| item |  |  | Required |
+| type | Can be<br/>- File<br/>- IP |  | Required |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![XDR Retro Hunt - Query](Insert the link to your image here)
```

Comment:
please generate the image and update the link here

Reply:
done